### ISSUE TITLE
Swelling of inguinal region

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10121]
**Preferred term label:**
Swelling of inguinal region

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0476313

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Inguinal hernia
 Femoral hernia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000913
- label: Swelling of inguinal region
- parents: [Abnormality of the abdominal wall (HP:0004298)](https://hpo.jax.org/app/browse/term/HP:0004298)
- pmid: 
- def: Bulging in the area of the lower abdomen, just above the thigh and near the groin.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10121
- synonyms:
